test/test_torch.py

```diff
@@ -3763,6 +3763,28 @@ def test_tensor_shape_empty(self, device):
         self.assertEqual([(0, 1, 3, 0)], [z.shape for z in torch.split(x, 1, dim=0)])
         self.assertEqual([(0, 1, 3, 0)], [z.shape for z in torch.split(x, 0, dim=0)])
 
+    def test_split_deprecated_overloads(self, device):
+        x = torch.randn((10, 1, 3, 2), device=device)
+
+        # split_sizes_or_sections is deprecated in favour of split_sizes
+        self.assertEqual(
+            torch.split(x, split_size_or_sections=2, dim=0),
+            torch.split(x, split_size=2, dim=0),
+        )
+        self.assertEqual(
+            torch.split(x, split_size_or_sections=5),
+            torch.split(x, split_size=5),
+        )
+
+        self.assertEqual(
+            torch.split(x, split_size_or_sections=[6, 4], dim=0),
+            torch.split(x, split_size=[6, 4], dim=0),
```

Comment:
@khabinov PTAL. I've added backward-compatible overloads.

Reply:
Thanks for addressing the issue @peterbell10! This looks good from backwards compatibility perspective.

Reply:
@albanD I assume we're okay with introducing a new name `split_size` for the second argument.

Reply:
It was already there for the method and not for the function.
I think it is best to make these consistent yes and so introduce the missing overload for now.

After this has been deprecated for 2 versions, we will be able to remove the duplicate overload and this will be all clean!